potimeb/Champion_of_Korascian.lua

```diff
@@ -1,4 +1,17 @@
+--Champion of Korascian - p2 Plane of Time
+-- NPC 223123
+--earth p2
+
 function event_death_complete(e)
-	-- send a signal to the zone_status that I died
-	eq.signal(223097,3); -- NPC: zone_status
+	--signal phase_two_controller
+	eq.signal(223242,1);
+end
+
+function event_signal(e)
+	--signal from An Earthen Overseer
+	if e.signal == 1 then
+		e.self:SetSpecialAbility(25, 0);
+		e.self:SetSpecialAbility(24, 0);
+		e.self:SetBodyType(1, true);
```

Comment:
Should use `SpecialAbility.immune_aggro`, `SpecialAbility.immune_aggro_on` and `BT.Humanoid` here instead (I know we don't do this perfectly everywhere, but we should!)